app/src/main/java/org/mozilla/focus/contextmenu/ContextMenuCandidates.kt

```diff
@@ -23,8 +22,9 @@ object ContextMenuCandidates {
         appLinksUseCases: AppLinksUseCases,
         snackBarParentView: View,
         snackbarDelegate: ContextMenuCandidate.SnackbarDelegate = DefaultSnackbarDelegate()
-    ): List<ContextMenuCandidate> =
-        if (context.components.experiments.isMultiTabsEnabled) {
+    ): List<ContextMenuCandidate> {
+        val isMultiTabExperimentEnabled = FocusNimbus.features.tabs.value().isMultiTab
```

Comment:
Nice, using a local variable is preferred to calling `FocusNimbus.features.tabs.value().isMultiTab` all the time.

My preferred style would be get all the configuration out of Nimbus with one call to `value()`.

```kt
val config = FocusNimbus.features.onboarding.value()

if (config.isEnabled) {}

if (config.isCfrEnabled) {} 
```

A bit redundant for just one flag here though :)